Doc/library/subprocess.rst

```diff
@@ -584,7 +584,7 @@ Instances of the :class:`Popen` class have the following methods:
 .. method:: Popen.poll()
 
    Check if child process has terminated.  Set and return
-   :attr:`~Popen.returncode` attribute.
+   :attr:`~Popen.returncode` attribute. Otherwise, returns ``None``.
```

Comment:
Elsewhere `Popen.returncode` is already documented to be `None` when the process hasn't terminated. But it does not hurt to also make that clear here.